accepted/global-json-updates.md

```diff
@@ -0,0 +1,790 @@
+* User includes a feature that requires an SDK that corresponds to a SDK feature band or higher, and wants to run the best available tools.
+  * User adds `rollForward: latestMajor` and the lowest version (with patch) they want to a `global.json`
+* User wants to know what version of the SDK is being used.
+  * Not addressed in this proposal: `global.json` will still be a subtle, hidden and easily forgotten marker
+* User rarely wants the risk of running the unsupported scenario of multiple SDK versions in a solution build.
+  * Not addressed in this proposal: There is nothing that encourages users to place `global.json` in a logical location.
+* Hosted CI, including Azure DevOps and Kudu, need to provide a small number of SDKs.
+  * It will now be possible for users to specify `global.json` files in a way that can work with Azure DevOps (roll-forward across at least feature bands)
+  * We will work with Azure DevOps to determine the recommended `global.json` for this scenario.
+
+## Examples of usage
+
+This section lists a series of scenarios and following the corresponding `global.json`. Following this is a table that shows the SDK that would be selected with specific combinations of SDKs present on the machine.
+
+### 1. Explicitly state today's behavior when a `global.json` specifies a version
+
+The following `global.json` defines a specific version number, and to use a higher version if it is available.
+
+```json
+{
+  "sdk": {
+    "version": "2.2.100",
+    "ignorePreview": false,
+    "rollForward": "patch"
+  }
+}
+```
+
+The common `global.json` for this case is likely to be:
+
+```json
+{
+  "sdk": {
+    "version": "2.2.100"
+  }
+}
+```
+
+### 2. Explicitly state today's behavior when there is no `global.json`
+
+Today's behavior is that an exact match is used if found, and if not found, the highest patch above the current in the feature band is used.
+
+If there is no `global.json` that would logically be the same as the following.
+
+```json
+{
+  "sdk": {
+    "ignorePreview": false,
+    "rollForward": "latestMajor"
+  }
+}
+```
+
+### 3. Skip previews, otherwise as though there was no `global.json`
+
+If there is no `global.json` but the user does not want to use previews, that would logically be the same as the following:
+
+```json
+{
+  "sdk": {
+    "ignorePreview": true,
+    "rollForward": "latestMajor"
+  }
+}
+```
+
+The common `global.json` for this case is likely to be:
+
+```json
+{
+  "sdk": {
+    "ignorePreview": true,
+  }
+}
+```
+
+The user may include their preference for previews high in their directory structure, and then individual project can elect different behavior with a new `global.json`. This will continue to follow today's rules. In particular: if there is a `global.json` file for a different reason, like a custom SDK, the higher `global.json` will not be used. 
+
+### 4 Select highest available, higher than specified
```

Comment:
The term "higher than specified" is confusing. It makes it sound like there is a policy where a version higher than the one specified will be selected. From reading the doc, I don't think there is such a policy.